### DISCUSSION TITLE
Sidebar title according to post

### DISCUSSION THREAD

**Discussion by nipa-it** [hugo-themes/toha#608] [Q&A] [answered]
Dear users,

I would like to implement into the cards/post.html the translated name of the sidebar title the post belongs to.

How can I refer to the title of the sidebar?

**Comment by hossainemruz** [comment]
@nipa-it Can you please explain little more?

**Comment by nipa-it** [comment, marked as answer]
Hi @hossainemruz ,

sorry for disturbing you but I've just solved:
it was just needed to add `{{ .Parent.Params.Title }}` into `cards/post.html`.

Thank you for what you built, it is really inspiring and liked from the first sight
